graphql-dgs-codegen-core/src/main/kotlin/com/netflix/graphql/dgs/codegen/CodeGenCli.kt

```diff
@@ -59,6 +59,7 @@ class CodeGenCli : CliktCommand("Generate Java sources for SCHEMA file(s)") {
     private val shortProjectionNames by option("--short-projection-names").flag()
     private val generateInterfaceSetters by option("--generate-interface-setters").flag()
     private val generateDocs by option("--generate-docs").flag()
+    private val generateBitset by option("--generate-bitset", help = "Generate bitset").flag(default = false)
```

Comment:
we may want to find a better name for this. bitset is related to how the feature is implemented rather than what it does. What this feature does is allow the user to check whether a field is set or not. We could call it `--generate-isset-check` or something like this

Reply:
Agreed! I have updated this to `--generate-field-is-set`, derivative of your suggestion. wdyt?